modules/Sources/Features/ImportWallet/ImportWallet.swift

```diff
@@ -20,29 +20,30 @@ public struct ImportWallet: Reducer {
     
     public struct State: Equatable {
         @PresentationState public var alert: AlertState<Action.Alert>?
-        public var birthdayHeight = "".redacted
+        @BindingState public var importedSeedPhrase = ""
+        @BindingState public var birthdayHeight = ""
         public var birthdayHeightValue: RedactableBlockHeight?
-        public var importedSeedPhrase = "".redacted
         public var isValidMnemonic = false
         public var wordsCount = 0
         public var isValidNumberOfWords = false
         public var maxWordsCount = 0
         public var isValidForm: Bool {
             isValidMnemonic &&
-            (birthdayHeight.data.isEmpty ||
-            (!birthdayHeight.data.isEmpty && birthdayHeightValue != nil))
+            (birthdayHeight.isEmpty ||
```

Comment:
Consider adding a test for this that serves as documentation and red flag if a change break the expectations, since these kind of statements are really good vectors for form validation bugs in prod.